Formula/gnuradio.rb

```diff
@@ -115,10 +93,17 @@ def install
                             gr-blocks testing gr-pager gr-noaa gr-channels
                             gr-audio gr-fcd gr-vocoder gr-fec gr-digital
                             gr-dtv gr-atsc gr-trellis gr-zeromq]
+    if build.with? "python"
+      enabled_components << "python"
+      enabled_components << "gr-utils"
+      enabled_components << "grc" if build.with? "pygtk"
```

Comment:
But then if I build with ``--without-python --with-pygtk`` it will still build with python. In my case it skips python. What is right way?